src/Hypergrid/events.js

```diff
@@ -560,7 +560,7 @@ exports.mixin = {
                  */
                 function(mouseEvent) {
                 var isMouseDownCell = this.mouseDownState && this.mouseDownState.gridCell.equals(mouseEvent.gridCell);
-                if (isMouseDownCell && mouseEvent.mousePointInClickRect) {
+                if (isMouseDownCell) {
```

Comment:
we never really rely on the event "mousePointInClickRect"
we now removed the check and added two extra "mousePointInLeftClickRect" and "mousePointInRightClickRect"
and pass down to down stream.